ANSWERS.md

```diff
@@ -1,7 +1,15 @@
 <!-- Answers to the Short Answer Essay Questions go here -->
 
 1. In Mocha, what are the differences between `before`, `after`, `beforeEach`, and `afterEach`? When do they run? What are they used for?
+Before: Actions that happen before a block occurs
+After: Actions that happen after a block occurs
+They are used to set up testing
+
 
 2. What is the point of Test Driven Development? What do you personally think about this approach?
+It wants you to develope towards the testing criteria.
+
+Jimmy Arias's dad did not listen to TDD when he helped develop the modern tennis forehand. He broke every criteria thought to be essential to having a good forehand. Federer has a tennis grip that is suppose to be impossible to play on tour with as does Nadel but they have done pretty well with them. Let the developers develope organicly to the situation. 
 
 3. What is a `spy` in `sinon`? How do we use it to effectively test a `callback`?
+The spy will keep track of what a function is doing.

```

Comment:
Again, this may be true, however, your answer does not adequately address the question. See Sean's lecture from Friday: https://youtu.be/JoIuYivEzhU?t=47m7s